components/abstractions/src/test/java/com/microsoft/kiota/RequestInformationTest.java

```diff
@@ -64,8 +64,28 @@ public void SetsPathParametersOfDateTimeOffsetType()
 
         // Assert
         var uriResult = assertDoesNotThrow(() -> requestInfo.getUri());
-        assertTrue(uriResult.toString().contains("fromDateTime='2022-08-01T00%3A00Z'"));
```

Comment:
I'm convinced that the existing check was not correct and the new one is better following the spec.